backend/app/routes/albums.py

```
@@ -91,6 +99,90 @@ def create_album(body: CreateAlbumRequest):
         )
 
 
+# POST /albums/from-memory - Create an album from a curated memory
+@router.post("/from-memory", response_model=CreateAlbumFromMemoryResponse)
+def create_album_from_memory(body: CreateAlbumFromMemoryRequest = Body(...)):
```

Ruff (0.16.0)

[warning] Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)